File: 7-kyu/even-or-odd---which-is-greater-/python/solution.py
def even_or_odd(s):
    even_sum = sum(int(n) for n in s if int(n) % 2 == 0)
    odd_sum = sum(int(n) for n in s if int(n) % 2 == 1)
    return 'Even is greater than Odd' if even_sum > odd_sum else 'Even and Odd are the same' if even_sum == odd_sum else 'Odd is greater than Even'

File: 7-kyu/even-or-odd---which-is-greater-/javascript/solution.js
const evenOrOdd = str => {
  const strInt = [...str].map((elem) => parseInt(elem));
  const evenSum = strInt.filter((elem) => elem % 2 === 0).reduce((acc, nxt) => acc + nxt);
  const oddSum = strInt.filter((elem) => elem % 2 === 1).reduce((acc, nxt) => acc + nxt);
  return evenSum > oddSum ? "Even is greater than Odd" : evenSum === oddSum ? "Even and Odd are the same" : "Odd is greater than Even";
};
